lib/kino/process.ex

```diff
@@ -740,13 +771,35 @@ defmodule Kino.Process do
 
     display =
       case process_info(pid, :registered_name) do
-        {:registered_name, []} -> inspect(pid)
-        {:registered_name, name} -> module_or_atom_to_string(name)
+        {:registered_name, []} ->
+          inspect(pid)
```

Comment:
✂️

Reply:
Ahhh, forgot to delet the `inspect`, tks.